storybook/src/alert/Modal.stories.tsx

```diff
@@ -43,6 +46,8 @@ export default {
 };
 
 export const Simple = () => {
+  // incl-code
+  // hook initialization
```

Comment:
Formatting errors (no whitespace in the beginning of code) are not fixable since Storybook trims the final outputted code that is returned from `transformSource`. Will remove the comments and leave it as is then (with no whitespace in the first row).